src/partials/smiley.hbs

```diff
@@ -0,0 +1,181 @@
+                    <!--[if mso | IE]>
+                    <div style="display:none;">
+                    <![endif]-->
+                    {{minLegend}}
+                    <!--[if mso | IE]>
+                    </div>
+                    <![endif]-->
+                  </div>
+                </td>
+              </tr>
+            </table>
+          </div>
+          <!--[if mso | IE]>
+              </td>
+            </tr>
+          </table>
+          <![endif]-->
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+<!--[if mso | IE]>
+        </td>
+      </tr>
+    </table>
+  </td>
+</tr>
+<tr>
+  <td class="" width="580px">
+    <table align="center" border="0" cellpadding="0" cellspacing="0" class="" style="width:580px;" width="580">
+      <tr>
+        <td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;">
+<![endif]-->
+<div style="margin:0px auto;max-width:580px;">
+  <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="width:100%;">
+    <tbody>
+      <tr>
+        <td style="direction:{{direction}};font-size:0px;padding:0px;text-align:center;">
+          <!--[if mso | IE]>
+          <table role="presentation" border="0" cellpadding="0" cellspacing="0">
+            <tr>
+          <![endif]-->
+          {{#each emojis}}
+          <!--[if mso | IE]>
+          <td class="" style="vertical-align:top;width:{{../width}}px;">
+          <![endif]-->
+          <div class="mj-column-per-100 mj-outlook-group-fix" style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:{{../width}}px;">
+            <table border="0" cellpadding="0" cellspacing="0" role="presentation" width="100%">
+              <tbody>
+                <tr>
+                  <td style="vertical-align:top;padding:1px 2px;">
+                    <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="" width="100%">
+                      <tr>
+                        <td align="center" style="font-size:0px;padding:0px;word-break:break-word;">
+                          <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="border-collapse:separate;width:100%;line-height:100%;">
+                            <tr>
+                              <td align="center" bgcolor="{{../colors.primary}}" role="presentation" style="border:none;border-radius:3px;cursor:auto;height:40px;mso-padding-alt:0px;background:{{../colors.primary}};"
+                                valign="middle"> <a {{#if url}} href="{{{url}}}" {{else}} sm-preview-value="{{value}}" {{/if}} style="display:block;background:{{../colors.primary}};color:{{../colors.background}};font-family:arial, helvetica, sans-serif;font-size:14px;font-weight:normal;line-height:1;margin:0;text-decoration:none;text-transform:none;padding:10px;mso-padding-alt:0px;border-radius:3px;"
```

Comment:
> `sm-preview-value="{{value}}"`

@E-Mud what does it do?

Reply:
@ksvitkovsky It's for previews, so when you click on the link the preview handler knows the underlying value of the choice.

Here you can see how it's used https://github.com/satismeter/satismeter/pull/878/files#diff-128684005375f32d91d39966ca0d1d964bc66b4f1b531675cca6354d45df9b5fR105